test/runtests.jl

```diff
@@ -7,7 +7,7 @@ const is_APPVEYOR = Sys.iswindows() && haskey(ENV, "APPVEYOR")
 
 @time begin
 
-if GROUP == "All" || GROUP == "Interface"
+if GROUP == "All" || GROUP == "Core" || GROUP == "Downstream" || GROUP == "Interface"
```

Comment:
no test should ever be multi-group like this.

Reply:
It would be nice if there was a design document that detailed things like this.
Currently,  you can not know based on this repo alone where some values of the environment variable, like Interface or Downstream will come from.

In any case, some CI in this repo passes trivially in a couple of microseconds because the environment variables are not set properly. This PR fixes it, but is probably too broad.